qiskit/circuit/library/blueprintcircuit.py

```diff
@@ -90,7 +90,7 @@ def data(self):
             self._build()
         return super().data
 
-    def qasm(self, formatted=False, filename=None):
+    def qasm(self, formatted=False, filename=None, encoding='utf-8'):
         if self._data is None:
             self._build()
         return super().qasm(formatted, filename)
```

Comment:
You need to include the encoding parameter here too. Otherwise this will be dropped if it's set to something other than utf8.

Reply:
Thank you, I though that I did this, but it appears, that I didn't commit that line.